### DISCUSSION TITLE
Shadow parameters

### DISCUSSION THREAD

**Discussion by danilokrulj** [google/model-viewer#4322] [Q&A]
Hi, I am trying to make something in three.js and I can't seem to get the shadow nearly as good as model viewer does. I figured since model-viewer uses three.js in the background I should try and ask here for some help or pointers. Does mv use PCFSoftShadows? What parameters do you guys set for the light source (I'm guessing its a directional light)?

**Comment by elalish** [comment]
No light source - we consider our shadows to be more akin to ambient occlusion of the ground. The code to create them is here: https://github.com/google/model-viewer/blob/master/packages/model-viewer/src/three-components/Shadow.ts
